app/Helpers/StringHelpers.php

```diff
@@ -287,4 +287,15 @@ public static function summation(array $array = [])
                 return implode(', ', $array) . ', and ' . $last;
         }
     }
+
+    public static function pageBlades($page)
```

Comment:
It was a little confusing to figure out what was going on in this function. Could you leave a comment that describes what it's doing and why?

Reply:
@zachgarwood updated the function. Idk why I wrote it "upside-down" I think my brain was working from the top of what the controller was doing when finding forTypes lol